### PR TITLE
Cp controllers

### PR DESCRIPTION
**Pull request by c-phillips7**
## What was done:

- Merged graces features/scheduler
- Fixed models and seeds so that db can be populated

## How to test:

- Run on browser

## Next steps:

- Transition Grace's feature to use our database rather than the example Paining/Gallery database

### CODE REVIEW

File: controllers/schedule-routes.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+const router = require('express').Router();
+const sequelize = require('../config/connection');
+const { Exercise, User, Workout, WorkoutExercise } = require('../models');
+const withAuth = require('../utils/auth');
+
+// GET schedule page
+// Use the custom middleware before allowing the user to access the scheduler
+router.get('/schedule', withAuth, async (req, res) => {
```
Comment (mavisyupyup1) [inline controllers/schedule-routes.js]: Maybe redundant(this is my work )
👍

Comment (c-phillips7) [reply controllers/schedule-routes.js]: Ok, I will get rid of it !

File: models/Exercise.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ Exercise.init(
       allowNull: false,
     },
     description: {
-      type: DataTypes.STRING,
+      type: DataTypes.TEXT,
```
Comment (mavisyupyup1) [inline models/Exercise.js:28]: Fantastic!! ❇️

File: models/Workout.js [hunk 3]
```diff
@@ -16,16 +16,8 @@ Workout.init(
       allowNull: false,
     },
     description: {
-      type: DataTypes.STRING,
+      type: DataTypes.TEXT,
```
Comment (mavisyupyup1) [inline models/Workout.js:19]: Maybe not necessary for this table. let me know what you think!

File: seeds/userData.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+const { User } = require('../models');
+
+const userData = [
+  {
+    user_id: 1,
```
Comment (mavisyupyup1) [inline seeds/userData.js:5]: needs to seed id rather than user_id. cc: @ronerlih

**Review by mavisyupyup1 — APPROVED** [review]
Fantastic work,Chris!! Thank you Ron!!!!!!

**Pull request merged**